### DISCUSSION TITLE
I cannot sell MAX of coins received or diamonds given in DUST

### DISCUSSION THREAD

**Discussion by omicreativedev** [deso-protocol/dips#82] [Q&A]
I have a ton of dust coin in my account. Coins received I can't sell MAX of. Coins I've bought that leave a residual amount behind that will always be in my wallet. Diamond dust. What's up with that?

**Comment by tijno** [comment]
Could you post an example screenshot - eg what kind of dust amounts are you having trouble with?

**Reply by bloated-devish** [reply]
Were these dusts given to you long time ago? or are these recent coins that you received?

**Comment by tijno** [comment]
@teambrixmusic could you show example of some of the dust?

**Comment by omicreativedev** [comment]
Mostly in Diamonds Received and Coins Received, any of the microsmall coin cannot be sold. If you hit max, the review button remains greyed out. I'll been sending these to another account I have just to clear my wallet of them. But it might be that they're not worth anything, Idk. But they remain in the wallet forever. Unless I'm doing something wrong.

As a workaround perhaps I could buy some coin then sell it in a chunk? Haven't tried that though.

![dust](https://user-images.githubusercontent.com/82254764/126076190-242b328b-d112-4d29-99bc-ec18936b362c.jpg)
